Documentation/network/concepts/ipam/cluster-pool.rst

```diff
@@ -45,6 +45,12 @@ Configuration
 For a practical tutorial on how to enable this mode in Cilium, see
 :ref:`gsg_ipam_crd_cluster_pool`.
 
+.. warning::
+    An existing element of the ``clusterPoolIPv4PodCIDRList`` list shouldn't be removed or changed. 
+    Doing so will cause unexpected behaviour. If you face a situation where the pool is exhausted,
+    add a new element to the list. The minimum mask length is ``/30``, but a mask of at least ``/29`` is recommended.
```

Comment:
I think we should document the reason why that is:

The first and last IP address in each pod CIDR block is reserved for the network and broadcast address. This restriction might be lifted in a future release.

Reply:
Sure, I can easily add that. Do you think a warning is still the right way of documenting it in the docs, or shall I change it to "not a warning".

Reply:
done